src/asymptotics.jl

```diff
@@ -0,0 +1,97 @@
+function _α_αp_asymptotic(v, x::Float64)
+    xinv = inv(x)^2
+    μ = 4 * v^2
+    s0 = 1
+    s1 = (1 - μ) / 8
+    s2 = evalpoly(μ, (-0.1953125, 0.203125, -0.0078125))
+    s3 = evalpoly(μ, (1.0478515625, -1.1591796875, 0.1123046875, -0.0009765625))
+    s4 = evalpoly(μ, (-11.466461181640625, 13.1358642578125, -1.71624755859375, 0.0469970703125, -0.000152587890625))
+    s5 = evalpoly(μ, (211.27614974975586, -246.8455924987793, 37.067405700683594, -1.5151596069335938, 0.017223358154296875, -2.6702880859375e-5))
+    s6 = evalpoly(μ, (-5892.0457146167755, 6965.548173427582, -1128.6143367290497, 56.11658954620361, -1.0105445384979248, 0.005837917327880859, -5.0067901611328125e-6))
+
+    if x < 18*v
```

Comment:
Need to figure out better way to select for how many terms here.
- 30 terms gives ~5e-16 relative error when x > 1.6*nu
- 20 terms gives ~2e-16 relative error when x > 2*nu
- 15 terms gives ~5e-18 relative error when x > 3*nu
- 10 terms gives ~3e-18 relative error when x > 5*nu